odtp/cli/setup.py

```diff
@@ -17,6 +18,8 @@ def initiate():
     odtpS3.create_folders(["odtp"])
     odtpS3.close()
 
+    create_folders()
```

Comment:
I struggle to understand this PR: you add two calls `create_folder` and `delete_folder`, but then the functions you add are `create_main_folders` and `delete_main_folders`. 

**suggestion**: I would rename these function to make it clearer that they belong to the filesystem setup. But then also what are the implications of these folders: if they are there the working directory in the dashboard should also be adjusted, but a path `user` is not very helpful in that regard: maybe there it would be better to have an `ODTP_PATH/github-username` home directory that is set for the user. 

I am not yet convinced about this PR. I don't think it is necessary for the `docker-compose` case and for the server setup I would leave this to the server administrator.